### PR TITLE
FB - MAJ migration site web to OVHcloud

### PR DESCRIPTION
**Pull request by fabienOVH**

### CODE REVIEW

**Comment by OVH-Team-Guides** [comment]
Please stick to the current format for guides links `/pages/link/to/the/guide`.
Also revert the deletion of anchor links, they are essential for translation purposes.

**Comment by Kazarus4** [comment]
Anchor links must be replaced by the legacy anchor links. 
Snippet doesn't respected for Title and excerpt.

File: pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
-title: Migrer son site et ses e-mails vers OVHcloud
-excerpt: Découvrez comment migrer votre site internet, vos e-mails et votre nom de domaine chez OVHcloud sans interruption de services
-updated: 2023-11-15
+title: Migrer son site web et ses services associés vers OVHcloud
+excerpt: Découvrez comment migrer l'intégralité de votre site web, incluant vos e-mails, vos domaines, votre base de données, chez OVHcloud sans interruption de services
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md]:
```suggestion
excerpt: Découvrez comment migrer votre site web, votre nom de domaine, votre base de données et vos e-mails chez OVHcloud sans interruption de services
```

File: pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md [hunk 2]
```diff
@@ -15,11 +15,16 @@ Ce guide vous présente les différentes actions à réaliser pour migrer l'ense
 > Nous mettons à votre disposition ce guide afin de vous accompagner au mieux sur des tâches courantes. Néanmoins, nous vous recommandons de faire appel à un [prestataire spécialisé](/links/partner) si vous éprouvez des difficultés. En effet, nous ne serons pas en mesure de vous fournir une assistance. Plus d'informations dans la section [« Aller plus loin »](#go-further) de ce guide.
 >
 
+> [!warning]
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md]:
```suggestion
```

File: pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md [hunk 3]
```diff
@@ -15,11 +15,16 @@ Ce guide vous présente les différentes actions à réaliser pour migrer l'ense
...
+>
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md]:
```suggestion
```

File: pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md [hunk 4]
```diff
@@ -15,11 +15,16 @@ Ce guide vous présente les différentes actions à réaliser pour migrer l'ense
...
+Ce guide s’adresse aux personnes possédant les bases dans les domaines tels que les [zones DNS](/pages/web_cloud/domains/dns_zone_general_information){.external}, les espaces de stockage FTP ou encore les bases de données. En revanche, vous n'avez pas besoin de posséder de connaissances techniques avancées, et ce guide vous aide pas à pas dans la migration de votre site web.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md]:
```suggestion
```

Comment (Kazarus4) [reply pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md]: fait doublon avec le paragraphe au dessus

File: pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md [hunk 5]
```diff
@@ -15,11 +15,16 @@ Ce guide vous présente les différentes actions à réaliser pour migrer l'ense
...
+>
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md]:
```suggestion
```

File: pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md [hunk 6]
```diff
@@ -15,11 +15,16 @@ Ce guide vous présente les différentes actions à réaliser pour migrer l'ense
...
+
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md]:
```suggestion
```

File: pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md [hunk 7]
```diff
@@ -52,16 +57,20 @@ Si une telle interruption devait avoir lieu, ce guide est construit de telle sor
 
 ### Etape 1 : commander l'hébergement et les adresses e-mail chez OVHcloud <a name="step1"></a>
 
-Plusieurs offres d'hébergement mutualisé OVHcloud contiennent une offre e-mail « MX Plan ». Cette offre e-mail permet de créer des adresses e-mail avec un espace de stockage de 5 Go maximum pour chaque adresse. Choisissez parmi les offres d'hébergement ci-dessous en fonction de la version PHP, de la version SQL, du nombre d'adresses e-mail dont vous avez besoin et de la taille de votre site à migrer :
+Plusieurs [offres d'hébergement mutualisé OVHcloud](https://www.ovhcloud.com/fr/web-hosting/){.external} contiennent une offre e-mail « [MX Plan](/pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_generalities){.external} ». Cette offre e-mail permet de créer des adresses e-mail avec un espace de stockage de 5 Go maximum pour chaque adresse. Choisissez parmi les offres d'hébergement ci-dessous en fonction de la version PHP, de la version SQL, du nombre d'adresses e-mail dont vous avez besoin et de la taille de votre site web à migrer :
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md]:
```suggestion
Plusieurs [offres d'hébergement mutualisé OVHcloud](/links/web/hosting) contiennent une offre e-mail « [MX Plan](/pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_generalities) ». Cette offre e-mail permet de créer des adresses e-mail avec un espace de stockage de 5 Go maximum pour chaque adresse. Choisissez parmi les offres d'hébergement ci-dessous en fonction de la version PHP, de la version SQL, du nombre d'adresses e-mail dont vous avez besoin et de la taille de votre site web à migrer :
```

File: pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md [hunk 8]
```diff
@@ -72,40 +81,48 @@ Une fois le paiement validé, l'installation de l'hébergement va démarrer. Un
...
+|---|---|---|---|
+|domain.tld.|MX|1|mx1.mail.ovh.net.|
+|domain.tld.|MX|5|mx2.mail.ovh.net.|
+|domain.tld.|MX|100|mx3.mail.ovh.net.|
+|www.domain.tld.|CNAME|-|domain.tld.|
+|domain.tld.|A|-|<adresse_IP_cible>|
+
+Pour récupérer la bonne adresse IP cible de votre hébergement OVHcloud, consultez notre guide listant les [adresses IP des différents clusters d'hébergement mutualisés](/pages/web_cloud/web_hosting/clusters_and_shared_hosting_IP).
+
+**Exemple** : Pour le nom de domaine « domain.tld », le rendu des entrées de votre domaine doit être le suivant :
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md]:
```suggestion
**Exemple** : Pour le nom de domaine « domain.tld », le rendu des entrées de votre nom de domaine doit être le suivant :
```

File: pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md [hunk 9]
```diff
@@ -72,40 +81,48 @@ Une fois le paiement validé, l'installation de l'hébergement va démarrer. Un
...
 
 ![hosting](images/dashboard-mx-a-cname.png){.thumbnail}
 
 > [!success]
 >
-> Notez les deux valeurs cibles des deux premières entrées de type « NS ». Elles serviront lors de l'[étape 9](#step9) de ce guide.
->
-> Ces valeurs correspondent aux serveurs DNS associés à cette zone DNS pour votre nom de domaine.
+> Notez les deux valeurs cibles ayant pour type d'enregistrement « NS ». Ces valeurs, de type `dnsXX.ovh.net` et `nsXX.ovh.net`, correspondent aux serveurs DNS associés à cette zone DNS pour votre nom de domaine. Elles serviront lors de l'[étape 9](#etape-9--remplacer-les-serveurs-dns-actifs-de-votre-nom-de-domaine-par-ceux-dovhcloud) de ce guide.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md]:
```suggestion
> Notez les deux valeurs cibles ayant pour type d'enregistrement « NS ». Ces valeurs, de type `dnsXX.ovh.net` et `nsXX.ovh.net`, correspondent aux serveurs DNS associés à cette zone DNS pour votre nom de domaine. Elles serviront lors de l'[étape 9](#step9) de ce guide.
```

File: pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md [hunk 10]
```diff
@@ -72,40 +81,48 @@ Une fois le paiement validé, l'installation de l'hébergement va démarrer. Un
...
 
-### Etape 4 : importer la sauvegarde de votre site Web sur votre offre d'hébergement OVHcloud <a name="step4"></a>
+### Etape 4 : importer la sauvegarde de votre site web sur votre offre d'hébergement OVHcloud <a name="step4"></a>
 
 Pour importer la sauvegarde de l'espace de stockage FTP de votre ancien prestataire, [connectez-vous à l'espace de stockage FTP de votre hébergement OVHcloud](/pages/web_cloud/web_hosting/ftp_connection) et téléversez la sauvegarde dans le dossier racine « www » (ou dans un autre dossier racine que vous aurez préalablement créé).
 
 Nous vous recommandons d'utiliser le logiciel [FileZilla](/pages/web_cloud/web_hosting/ftp_filezilla_user_guide) pour téléverser votre sauvegarde FTP sur votre hébergement.
 
-Si votre fichier de sauvegarde est compressé (zippé), décompressez-le dans un dossier vide sur votre ordinateur avant de téléverser vos fichiers sur l'hébergement OVHcloud.
+Si votre fichier de sauvegarde est compressé (zippé), décompressez-le dans un dossier vide sur votre ordinateur avant de téléverser l'ensemble des fichiers sur l'hébergement OVHcloud.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md]:
```suggestion
Si votre fichier de sauvegarde est compressé (zippé), décompressez-le dans un dossier vide sur votre ordinateur avant de téléverser vos fichiers sur l'hébergement OVHcloud.
```

File: pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md [hunk 11]
```diff
@@ -213,9 +230,9 @@ Une fois vos anciennes adresses e-mail migrées chez OVHcloud, reconfigurez vos
 
 ### Etape 9 : remplacer les serveurs DNS actifs de votre nom de domaine par ceux d'OVHcloud <a name="step9"></a>
 
-La zone DNS pré-configurée lors de l'[étape 2](#step2) n'est pas encore appliquée à votre nom de domaine.
+La zone DNS pré-configurée lors de l'[étape 2](#etape-2--créer-et-préconfigurer-une-zone-dns-pour-votre-nom-de-domaine-chez-ovhcloud) n'est pas encore appliquée à votre nom de domaine. Actuellement, votre domaine utilise toujours les serveurs DNS de votre fournisseur d'origine.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md]:
```suggestion
La zone DNS pré-configurée lors de l'[étape 2](#step2) n'est pas encore appliquée à votre nom de domaine. Actuellement, votre nom de domaine utilise toujours les serveurs DNS de votre fournisseur d'origine.
```

File: pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md [hunk 12]
```diff
@@ -213,9 +230,9 @@ Une fois vos anciennes adresses e-mail migrées chez OVHcloud, reconfigurez vos
...
 
-Remplacez les serveurs DNS actuels de votre nom de domaine par les deux serveurs DNS déclarés dans la zone DNS OVHcloud.
+Remplacez les serveurs DNS actuels (du registrar d'origine) par les deux serveurs DNS déclarés dans la zone DNS OVHcloud (de type `dnsXX.ovh.net` et `nsXX.ovh.net`). Cette manipulation se fait dans l'interface de gestion du registrar d'origine.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md]:
```suggestion
Remplacez les serveurs DNS actuels (du registrar d'origine) par les deux serveurs DNS déclarés dans la zone DNS OVHcloud (de type `dnsXX.ovh.net` et `nsXX.ovh.net` ou `dns200.anycast.me` et `ns200.anycast.me`). Cette manipulation se fait dans l'interface de gestion du registrar d'origine.
```

File: pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md [hunk 13]
```diff
@@ -224,14 +241,18 @@ Remplacez les serveurs DNS actuels de votre nom de domaine par les deux serveurs
 
 ### Etape 10 : transférer votre nom de domaine chez OVHcloud <a name="step10"></a>
 
-Une fois la propagation DNS terminée, testez votre site et vérifiez l'envoi et la réception des e-mails depuis vos adresses e-mail.
+Une fois la propagation DNS terminée, vérifiez que l'ensemble de votre site web est fonctionnel. Naviguez sur votre site internet pour vérifier que toutes les pages s'affichent correctement et qu'aucune erreur 404 ne soit renvoyée. Vérifiez également l'envoi et la réception des e-mails depuis vos adresses e-mail.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md]:
```suggestion
Une fois la propagation DNS terminée, vérifiez que l'ensemble de votre site web est fonctionnel. Naviguez sur votre site internet pour vérifier que toutes les pages s'affichent correctement et qu'aucune erreur 404 ne soit renvoyée. Vérifiez également l'envoi et la réception des e-mails depuis vos adresses e-mail.

```

File: pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md [hunk 14]
```diff
@@ -52,16 +52,20 @@ Si une telle interruption devait avoir lieu, ce guide est construit de telle sor
...
 
 - L'hébergement [Perso](/links/web/hosting-personal-offer) avec **10 adresses e-mail** « MX Plan »
 - L'hébergement [Pro](/links/web/hosting-professional-offer) avec **100 adresses e-mail** « MX Plan »
 - L'hébergement [Performance](/links/web/hosting-performance-offer) avec **1000 adresses e-mail** « MX Plan ». Cette offre est déclinée en 4 « sous-offres ».
 - L'hébergement [Cloud Web](/links/web/hosting-cloud-web-offer) avec **200 adresses e-mail** « MX Plan ». Cette offre est utilisée par les développeurs d'applications.
 
-Une fois votre offre d'hébergement choisie, si vous n'êtes pas encore client OVHcloud, cliquez sur le bouton `Commander`{.action} présent sur les pages commerciales ci-dessus. Suivez les étapes de la commande **sans demander le transfert de votre nom de domaine**.
+Une fois votre offre d'hébergement choisie, si vous n'êtes pas encore client OVHcloud, cliquez sur le bouton `Commander`{.action} présent sur les pages commerciales ci-dessus. Suivez les étapes de la commande **sans demander le transfert de votre nom de domaine** (Cette action sera effectuée à l'étape 10 de ce guide).
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md]:
```suggestion
Une fois votre offre d'hébergement choisie, si vous n'êtes pas encore client OVHcloud, cliquez sur le bouton `Commander`{.action} présent sur les pages commerciales ci-dessus. Suivez les étapes de la commande **sans demander le transfert de votre nom de domaine** (cette action sera effectuée à l'étape 10 de ce guide).
```

File: pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md [hunk 15]
```diff
@@ -52,16 +52,20 @@ Si une telle interruption devait avoir lieu, ce guide est construit de telle sor
...
 
-Vous pouvez aussi effectuer la commande depuis votre [espace client OVHcloud](/links/manager). Une fois connecté, cliquez sur la partie `Web Cloud`{.action}, cliquez ensuite sur le bouton `Commander`{.action}, dans la partie supérieure gauche, puis sur `Hébergement`{.action}. Poursuivez les étapes de la commande **sans demander le transfert de votre nom de domaine**.
+Vous pouvez aussi effectuer la commande depuis votre [espace client OVHcloud](/links/manager). Une fois connecté, suivez les instructions suivantes :
+
+- Rendez-vous sur l'onglet `Web Cloud`{.action}.
+- En haut à gauche de l'interface, cliquez sur le bouton `Commander`{.action} puis sur `Hébergement`{.action}.
+- Poursuivez les étapes de la commande **sans demander le transfert de votre nom de domaine** (Cette action sera effectuée à l'étape 10 de ce guide).
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md]:
```suggestion
- Poursuivez les étapes de la commande **sans demander le transfert de votre nom de domaine** (cette action sera effectuée à l'étape 10 de ce guide).
```

File: pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md [hunk 16]
```diff
@@ -72,34 +76,42 @@ Une fois le paiement validé, l'installation de l'hébergement va démarrer. Un
 
 ### Etape 2 : créer et préconfigurer une zone DNS pour votre nom de domaine chez OVHcloud <a name="step2"></a>
 
+Si votre domaine se trouve chez un autre prestataire et que vous souhaitez le transférer chez OVHcloud, vous devez dans un premier temps créer et pré-configurer une zone DNS avant d'initier le transfert, afin d'éviter l'interruption de services.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md]:
```suggestion
Si votre domaine se trouve chez un autre prestataire et que vous souhaitez le transférer chez OVHcloud, vous devez dans un premier temps créer et pré-configurer une zone DNS avant d'initier le transfert, afin d'éviter une interruption de services.
```

File: pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md [hunk 17]
```diff
@@ -72,34 +76,42 @@ Une fois le paiement validé, l'installation de l'hébergement va démarrer. Un
...
+
 Lorsque votre hébergement est créé, connectez-vous à votre [espace client OVHcloud](/links/manager) puis créez une zone DNS pour votre nom de domaine **sans les « www »**. Vous pouvez vous aider de notre guide sur la [création d'une zone DNS chez OVHcloud](/pages/web_cloud/domains/dns_zone_create).
 
-Une fois la zone DNS créée, accédez à sa gestion à l'aide de notre guide « [Éditer une zone DNS OVHcloud](/pages/web_cloud/domains/dns_zone_edit) ». Si elles ne sont pas présentes, renseignez les entrées suivantes :
+Une fois la zone DNS créée, accédez à sa gestion à l'aide de notre guide « [Éditer une zone DNS OVHcloud](/pages/web_cloud/domains/dns_zone_edit) ».
+
+Si elles ne sont pas présentes, renseignez les entrées suivantes :
+
+**Exemple** : Pour le nom de domaine « domain.tld »
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md]:
```suggestion
**Exemple**  (pour le nom de domaine « domain.tld ») :
```

File: pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md [hunk 18]
```diff
@@ -224,14 +236,19 @@ Remplacez les serveurs DNS actuels de votre nom de domaine par les deux serveurs
 
 ### Etape 10 : transférer votre nom de domaine chez OVHcloud <a name="step10"></a>
 
-Une fois la propagation DNS terminée, testez votre site et vérifiez l'envoi et la réception des e-mails depuis vos adresses e-mail.
+Une fois la propagation DNS terminée, vérifiez que l'ensemble de votre site web est fonctionnel. Naviguez sur votre site internet pour vérifier que toutes les pages s'affichent correctement et qu'aucune erreur 404 ne soit renvoyée. Vérifiez également l'envoi et la réception des e-mails depuis vos adresses e-mail.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/hosting_migrating_to_ovh/guide.fr-fr.md]:
```suggestion
Une fois la propagation DNS terminée, vérifiez que l'ensemble de votre site web est fonctionnel. Naviguez sur votre site internet pour vérifier que toutes les pages s'affichent correctement et qu'aucune erreur 404 n'est renvoyée. Vérifiez également l'envoi et la réception des e-mails depuis vos adresses e-mail.
```

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @fabienOVH

**Pull request merged**